frontend/src/pages/notebookController/screens/admin/NotebookAdminContext.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Notebook } from 'types';
+import { User } from './types';
+
+type NotebookAdminContextProps = {
+  setNotebooksToStop: (notebooks: Notebook[]) => void;
+  notebooksToStop: Notebook[];
```

Comment:
it is unnecessary to hold onto this state if it's in your other state... you're just holding onto duplicate information